components/infra-proxy-service/server/servers_test.go

```diff
@@ -0,0 +1,519 @@
+				Fqdn:        "domain.com",
+				IpAddress:   "0.0.0.0",
+			}
+			resp2, err := cl.CreateServer(ctx, req2)
+			assert.Nil(t, resp2)
+			grpctest.AssertCode(t, codes.AlreadyExists, err)
+			cleanupServer(ctx, t, cl, resp1.Server.Id)
+		})
+
+		t.Run("when the server description is missing, raise invalid argument error", func(t *testing.T) {
+			resp, err := cl.CreateServer(ctx, &request.CreateServer{
+				Name:      "chef-infra-server",
+				Fqdn:      "domain.com",
+				IpAddress: "0.0.0.0",
+			})
+			assert.Nil(t, resp)
+			assert.Error(t, err, "must supply server description")
+			grpctest.AssertCode(t, codes.InvalidArgument, err)
+		})
+
+		t.Run("when the server fqdn is missing, raise invalid argument error", func(t *testing.T) {
+			resp, err := cl.CreateServer(ctx, &request.CreateServer{
+				Name:        "chef-infra-server",
+				Description: "Chef infra server",
+				IpAddress:   "0.0.0.0",
+			})
+			assert.Nil(t, resp)
+			assert.Error(t, err, "must supply server fqdn")
+			grpctest.AssertCode(t, codes.InvalidArgument, err)
+		})
+
+		t.Run("when the server IP address is missing, raise invalid argument error", func(t *testing.T) {
+			resp, err := cl.CreateServer(ctx, &request.CreateServer{
+				Name:        "chef-infra-server",
+				Description: "Chef infra server",
+				Fqdn:        "domain.com",
+			})
+			assert.Nil(t, resp)
+			assert.Error(t, err, "must supply server IP address")
+			grpctest.AssertCode(t, codes.InvalidArgument, err)
+		})
+	})
+
+	t.Run("GetServers", func(t *testing.T) {
+		test.ResetState(context.Background(), t, serviceRef)
+
+		t.Run("when there are no servers in db, return empty list", func(t *testing.T) {
+			list, err := cl.GetServers(ctx, &request.GetServers{})
+			require.NoError(t, err)
+			require.Nil(t, list.Servers)
+			assert.Equal(t, 0, len(list.Servers))
+		})
+
+		t.Run("when there are some servers in db, return all the servers successfully", func(t *testing.T) {
+			resp1, err := cl.CreateServer(ctx, &request.CreateServer{
+				Name:        "chef-infra-server1",
+				Description: "Chef infra server",
+				Fqdn:        "domain1.com",
+				IpAddress:   "10.0.0.1",
+			})
+			require.NoError(t, err)
+			require.NotNil(t, resp1)
+
+			resp2, err := cl.CreateServer(ctx, &request.CreateServer{
+				Name:        "chef-infra-server2",
+				Description: "Chef infra server",
+				Fqdn:        "domain2.com",
+				IpAddress:   "10.0.0.2",
+			})
+			require.NoError(t, err)
+			require.NotNil(t, resp2)
+
+			list, err := cl.GetServers(ctx, &request.GetServers{})
+			require.NoError(t, err)
+			require.NotNil(t, list)
+			assert.Contains(t, list.Servers, resp1.Server)
+			assert.Contains(t, list.Servers, resp2.Server)
+			assert.Equal(t, 2, len(list.Servers))
+
+			cleanupServer(ctx, t, cl, resp1.Server.Id)
+			cleanupServer(ctx, t, cl, resp2.Server.Id)
+		})
+
+		t.Run("when the server is exist with it's orgs, return servers list with orgs's count successfully", func(t *testing.T) {
```

Comment:
Slight cleanup; also the word "successfully" is already implied so not needed.
```suggestion
		t.Run("when the server exists with orgs, return servers list with org count", func(t *testing.T) {
```